README.md

```diff
@@ -1,97 +1,88 @@
 
-We have a Python scraper running on AWS and submitting predicted subway arrivals to an AWS PostgreSQL database since late February 2017. We want to process this data to generate observed station arrival times for each train at each station. Have a look at [**How to Get Involved**](#how-to-get-involved). There was a couple month hiatus in data scraping because the database was full between August and November, but the scraper is now continuing to hum along nicely.
+We're still trying to process the predicted arrival times obtained from the API into reasonably reliable state.  This work is happening in a Jupyter Notebook Filtering Observed Arrivals.ipynb; it is mostly in SQL, despite being in a Python notebook.
 
-We're still trying to process the predicted arrival times we get from the API into reasonably reliable observed arrivals. This work is happening in a Jupyter Notebook [Filtering Observed Arrivals.ipynb](doc/Filtering%20Observed%20Arrivals.ipynb) . The work is actually mostly in SQL, despite being in a Python notebook. Feel free to follow along. More eyes on the data leads to better data.
+Take a look at [How to Get Involved](#how-to-get-involved) with your expertise.  Feel free to follow along; your informed feedback will surely lead to better data.
 
-### First Step Metric
+## First Step Metric
+**Buffer Time** - A measure of reliability useful for the rider:
 
-**Buffer Time:** A measure of reliability useful for the customer: how much extra time a customer needs to buffer in order for their trip to be on time 1 day out of 20.
+- how much extra time he/she needs to buffer in order for their trip to be on time 1 trip out of 20.
 
 ## Documentation
-Have a look in the [`doc/`](doc/) folder for Jupyter Notebooks explaining how we explored developing this project and understanding the data.
+Review the [doc/](doc/) folder for the Jupyter Notebooks explaining how we explored developing this project and understanding the data.
 
-### Data Flow and Data Structure
-The scraper runs every minute when the TTC is in service. Each of these runs is logged in `polls`, with a unique `pollid` and a start and end time. 
-During one run of the scraper, each station gets its predicted arrivals requested. This is logged in `requests`, with a unique `requestid` and noting which station it is using `stationid` and `lineid`. 
-For each request, 3 predicted arrivals are recorded for each line and direction at that station. This is stored in `ntas_data` (Next Train Arrival System). This table notes the train's `traindirection` and its unique id `trainid`, the time until the train's arrival `timint` and a `train_message`, whether the train is arriving, at station, or is delayed.
-**For more info:** have a look at the API exploration notebook under [`doc/API_exploration.ipynb`](https://github.com/CivicTechTO/ttc_subway_times/blob/master/doc/API_exploration.ipynb)
```

Comment:
Red 17-21 to be deleted, but decide if "(https://github.com/CivicTechTO/ttc_subway_times/blob/master/doc/API_exploration.ipynb)" this should be reintegrated to the new 21 section. 
Also, modified the quoted blue terms into the new 21 section term by term.